hs_discover/package-lock.json

```diff
@@ -0,0 +1,11624 @@
+{
```

Comment:
This file is over 11000 lines long.  Are all of these packages necessary?  How was it generated?

Reply:
package-lock.json is automatically generated and is a standard npm file. It is best practice to commit to source code. https://docs.npmjs.com/configuring-npm/package-lock-json.html

Reply:
that's quite the dependency tree generated from such a short package.json file.  I ran it locally and it generated similar sized file to convince myself.